src/core/img/icons/magnifying-glass.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="56px" height="56px" viewBox="-4 25 56 56" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Ah, right, I forgot to. Yeah this one is right out of @pwalm's mock so I'm using it, but I'll delete the old one.